projects/1.20/assets/gag/gag/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+  "itemGroup.gag.gag": "GAG — Gadgets Against Grind",
+  "info.gag.repelling_item": "使用后，给予玩家一个阻止附近的敌对生物生成的效果。",
+  "info.gag.supports_unbreaking": "（可被耐久与经验修补附魔）",
+  "info.gag.time_sand_tooltip_mult": "此方块已加速%sx倍！",
+  "info.gag.time_sand_tooltip_time": "剩余时间：%s秒",
+  "item.gag.time_sand": "时间之沙",
+  "item.gag.time_sand_pouch": "时间之袋",
+  "item.gag.time_sand_pouch.info.stored_grains": "已存储%2$s的时间颗粒",
+  "item.gag.time_sand_pouch.info.1": "在玩家的物品栏中缓慢存储时间颗粒。",
+  "item.gag.time_sand_pouch.info.2": "右击工作中的方块，消耗袋中存储的时间颗粒以加速其运行。",
+  "item.gag.escape_rope": "逃生绳",
+  "item.gag.escape_rope.no_space": "无法在地面上找到合适的空间进行传送。",
+  "item.gag.escape_rope.info": "将玩家从洞穴或建筑物中传送到最近的地面上。",
+  "item.gag.hearthstone": "炉石",
+  "item.gag.energized_hearthstone": "充能炉石",
+  "item.gag.hearthstone.no_target": "找不到可以传送的地点，您设置了重生点吗？",
+  "item.gag.hearthstone.too_weak": "这块炉石似乎太弱了，无法进行传送。",
+  "item.gag.hearthstone.info": "将玩家传送至重生点，或传送至世界重生点。",
+  "item.gag.hearthstone.info_adv": "这块炉石已经聚集了闪电的能量,",
+  "item.gag.hearthstone.info_adv_2": "可以传送到任何一个被绑定的位置！（潜行状态下使用以绑定）",
+  "item.gag.hearthstone.info_adv_3": "请注意，如果需要重新绑定，您需要再次用闪电击中它。",
+  "item.gag.hearthstone.target.bound": "当前绑定于%s",
+  "item.gag.hearthstone.target.unbound": "当前未绑定",
+  "item.gag.hearthstone.target.respawn": "您的重生点",
+  "item.gag.hearthstone.warmup": "将于%s秒后传送...",
+  "item.gag.labeling_tool": "标记工具",
+  "item.gag.labeling_tool.info": "免费更改您物品栏中任何物品的名称。",
+  "item.gag.labeling_tool.charges": "剩余%s次",
+  "item.gag.labeling_tool.no_charges": "次数已用尽！",
+  "item.gag.labeling_tool.text_box": "重命名物品",
+  "item.gag.pigment_jar": "颜料罐",
+  "item.gag.pigment_jar.contents": "可使用%s次%s",
+  "item.gag.pigment_jar.contents.empty": "空",
+  "item.gag.pigment_jar.info.empty.1": "用于储存颜料的空罐子。可能需要装满才能使用。",
+  "item.gag.pigment_jar.info.empty.2": "提示：用燧石研磨染料，然后加入牛奶。",
+  "item.gag.pigment_jar.info.filled.1": "可以在标记工具中使用以改变物品名称的颜色。",
+  "item.gag.pigment_jar.info.filled.2": "提示：与其他颜料罐混合以调配颜色。",
+  "block.gag.no_solicitors": "‘游人止步！’告示牌",
+  "block.gag.no_solicitors.info.1": "在合理范围内驱逐某些讨厌的入侵者。",
+  "block.gag.no_solicitors.info.2": "（右键点击使用羊毛切换静音模式）",
+  "block.gag.no_solicitors.silent": "静音模式：%s",
+  "item.gag.sacred_salt": "神圣之盐",
+  "item.gag.sacred_salve": "神圣药膏",
+  "item.gag.sacred_balm": "神圣香油",
+  "item.gag.sacred_salve.extra": "比神圣之盐的持续时间稍长，作用范围也更大。",
+  "item.gag.sacred_balm.extra": "比神圣之盐或神圣药膏的持续时间要长得多，但作用范围减小了。",
+  "effect.gag.repelling": "驱散",
+  "item.gag.mining_dynamite": "采矿炸药",
+  "item.gag.mining_dynamite.info": "可投掷的爆炸物，可以高效地摧毁石头。被摧毁的所有方块都会掉落。",
+  "item.gag.fishing_dynamite": "捕鱼炸药",
+  "item.gag.fishing_dynamite.info": "只能在水下使用的可投掷爆炸物，对鱼类造成成倍伤害。如果您不喜欢钓鱼，这将是完美的选择！",
+  "commands.gag.give_time.success": "成功将%stick的时间给予%s",
+  "commands.gag.give_time.no_pouch": "%s的物品栏中没有时间之袋！",
+  "subtitles.gag.entity.dynamite.throw": "投掷的炸药",

```

字幕格式参见https://zh.minecraft.wiki/w/%E5%AD%97%E5%B9%95
```suggestion
  "subtitles.gag.entity.dynamite.throw": "炸药：飞出",
```